aspnetcore/host-and-deploy/health-checks.md

```diff
@@ -633,6 +635,45 @@ The `IHealthCheckPublisher` interface has a single method:
 Task PublishAsync(HealthReport report, CancellationToken cancellationToken);
 ```
 
+`HealthCheckPublisherOptions` allow you to set:
+
+* `Delay` &ndash; The initial delay applied after the app starts before executing `IHealthCheckPublisher` instances. The delay is applied once at startup and doesn't apply to subsequent iterations. The default value is five seconds.
+* `Period` &ndash; The period of `IHealthCheckPublisher` execution. The default value is 30 seconds.
+* `Predicate` &ndash; If `Predicate` is `null` (default), the health check publisher service runs all registered health checks. To run a subset of health checks, provide a function that filters the set of checks. The predicate is evaluated each period.
+* `Timeout` &ndash; The timeout for executing the health checks for all `IHealthCheckPublisher`instances. Use <xref:System.Threading.Timeout.InfiniteTimeSpan> to execute without a timeout. The default value is 30 seconds.
+
+::: moniker range="= aspnetcore-2.2"
+
+> [!WARNING]
+> Due to an issue in the ASP.NET Core 2.2 release, setting `Period` isn't honored by the `IHealthCheckPublisher` implementation and inadvertently sets the value of `Delay`. This issue will be fixed in ASP.NET Core 3.0. For more information, see [HealthCheckPublisherOptions.Period sets the value of .Delay](https://github.com/aspnet/Extensions/issues/1041).
```

Comment:
In the ASP.NET Core 2.2 release, setting `Period` isn't honored by the `IHealthCheckPublisher` implementation; it sets the value of `Delay`. This issue will be fixed in ASP.NET Core 3.0. For more information, see [HealthCheckPublisherOptions.Period sets the value of .Delay](https://github.com/aspnet/Extensions/issues/1041).